site/src/App.style.js

```diff
@@ -218,7 +218,7 @@ export const Main = styled.main`
 
   .caption {
     color: ${({ theme }) => theme.text};
-    margin-top: 0.75rem;
+    margin-top: 3rem;
```

Comment:
This is logically not needed, this is a caption and should be close to the image and not same as the callout text below it.